src/bridge/mgmtscripts/add_users.py

```diff
@@ -36,6 +36,7 @@
     sys.exit(1)
 
 userdata = {
+    'id' : 1,
```

Comment:
@noushi,

So I got around to testing this and the user id needs to be a string as the cookie creation expects that data to be a string rather than a integer. You can accomplish the same thing though by setting it to something like.

    'id' : 'supercoolusernumber1',